packages/mui-material/src/Autocomplete/Autocomplete.test.js

```diff
@@ -864,6 +831,50 @@ describe('<Autocomplete />', () => {
     expect(handleSubmit.callCount).to.equal(4);
   });
 
+  it('should not open the autocomplete popup when deleting chips', async () => {
+    const { queryByRole, queryByText, user } = render(
+      <Autocomplete
+        multiple
+        options={['one', 'two', 'three']}
+        defaultValue={['one']}
+        renderInput={(params) => <TextField {...params} autoFocus />}
+      />,
+    );
+
+    expect(queryByRole('listbox')).to.equal(null);
+
+    const chip = queryByText('one').parentElement;
+    expect(chip).not.to.equal(null);
+
+    // Delete the chip
+    await user.click(chip.getElementsByClassName(chipClasses.deleteIcon)[0]);
+
+    expect(queryByText('one')).to.equal(null);
+    expect(queryByRole('listbox')).to.equal(null);
+  });
+
+  it('should toggle the autocomplete popup when clicking the popup indicator', async () => {
+    const { queryByRole, getByTestId, user } = render(
+      <Autocomplete
+        multiple
+        options={['one', 'two', 'three']}
+        renderInput={(params) => <TextField {...params} autoFocus />}
+        slotProps={{ popupIndicator: { 'data-testid': 'popup-indicator' } }}
+      />,
+    );
+
+    expect(queryByRole('listbox')).to.equal(null);
+
+    const popupIndicator = getByTestId('popup-indicator');
```

Comment:
I think we could use something like this:

```suggestion
    const popupIndicator = getByRole('button', { name: 'Open' });
```

`"Open"` is the default value for the `openText` prop.

Reply:
Nice 👌🏼 updated